src/components/Book.js

```diff
@@ -1,18 +1,34 @@
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { REMOVE_BOOK } from '../actions';
 
 const Book = props => {
-  const { bookObject } = props;
+  const { bookObject, onDeleteBook } = props;
+
+  const handleClick = () => {
+    onDeleteBook(bookObject);
+  };
```

Comment:
- [ ] This method should be implemented in the `BookList` component, not here. This will improve the speed of your application since you won't have a function in each `Book` component. Implement it there and pass it down as a prop to each book([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)).